dev-docs/show-prebid-ads-on-amp-pages.md

```diff
@@ -10,335 +10,98 @@ nav_section: prebid-amp
-
 ### AMP content page
 
-The special meta tag in [amp_page.html](https://github.com/prebid/Prebid.js/blob/master/integrationExamples/gpt/amp/amp_page.html) specifies the path to the publisher-hosted file to use as the AMP cross-domain host:
-
-```html
-<meta name="amp-3p-iframe-src" content="https://amp.publisher.com:5000/remote.html">
-```
-
 The `amp-ad` elements in the page body need to be set up as shown below, especially the following attributes:
 
-+ *data-slot*: Identifies the ad slot for the auction.
-+ *json*: Used to pass JSON configuration data across to the cross-domain host running the Prebid auction.  The format of the JSON is as shown: a `"prebid"` object containing the standard `"adUnits"` you know and love, plus a `"requestBidsDuration"` key which sets the Prebid auction timeout (in milliseconds).
-
-```html
-    <amp-ad
-      width="300"
-      height="250"
-      layout="fixed"
-      type="doubleclick"
-      json='{"prebid":{"requestBidsDuration":1000,"adUnits":[{"code":"/19968336/header-bid-tag-1","sizes":[[300,250],[300,600],[300,250],[100,100]],"bids":[{"bidder":"appnexusAst","params":{"placementId":"10433394","dealId":"some deal!"}},{"bidder":"aol","params":{"network":"10077.1","placement":3671670}},{"bidder":"sovrn","params":{"tagid":"315045"}}]},{"code":"/19968336/header-bid-tag-2","sizes":[[300,250],[300,600],[300,250],[100,100]],"bids":[{"bidder":"appnexusAst","params":{"placementId":"10433394","dealId":"some deal!"}},{"bidder":"aol","params":{"network":"10077.1","placement":3671670}},{"bidder":"sovrn","params":{"tagid":"315045"}}]}]}}'
-      data-slot="/19968336/header-bid-tag-1">
-    </amp-ad>
-```
-
-
-<a name="prebid.js-page" />
-
-### Prebid.js page
-
-The instructions below try to explain the code in [remote.html](https://github.com/prebid/Prebid.js/tree/master/integrationExamples/gpt/amp/remote.html) in the [AMP integration example in our repo](https://github.com/prebid/Prebid.js/tree/master/integrationExamples/gpt/amp).  When in doubt, use the AMP example files in the repo as your primary reference.
-
-The steps below are written to correspond roughly with the order in which you might write the code.
-
-We recommend reading this side-by-side with [the AMP example files](https://github.com/prebid/Prebid.js/tree/master/integrationExamples/gpt/amp) to see exactly how it's all laid out.
-
-This file is where the Prebid auction actually runs.
-
-#### 1. Add the AMP project's async file loader
-
-In [remote.html](https://github.com/prebid/Prebid.js/tree/master/integrationExamples/gpt/amp/remote.html), load the `f.js` async file loader from the AMP project.  This is a standard method for loading 
++ `data-slot`: Identifies the ad slot for the auction.
++ `rtc-config`: Used to pass JSON configuration data to [Prebid Server][PBS], which will actually handle the communication with AMP RTC.  The `vendors` property should list all of the Prebid Server vendor instances you will be using.
```

Comment:
"The `rtc_config.vendors` property should define any vendors that will be receiving RTC callouts (including Prebid Server) up to a maximum of five."